tsconfig.json

```diff
@@ -1,35 +1,40 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "module": "esnext",
-    "moduleResolution": "node",
+    "target": "ESNext",
+    "module": "ESNext",
+    "moduleResolution": "NodeNext",
     "lib": ["es2017", "dom"],
     "allowJs": true,
     "checkJs": true,
-    "noEmit": true,
     "strict": true,
     "noImplicitThis": true,
+    "composite": true,
+    "emitDeclarationOnly": true,
     "alwaysStrict": true,
-    "types": ["node", "mocha", "sinon"],
+    "types": ["node", "mocha", "chai", "sinon"],
     "esModuleInterop": true,
     "suppressImplicitAnyIndexErrors": true,
-    "skipLibCheck": true
-  },
-  "include": ["packages/**/*.js", "packages-node/**/*.js"],
-  "exclude": [
-    "node_modules",
-    "**/node_modules/*",
-    "**/coverage/*",
-    "**/dist/**/*",
-    "packages/**/test-helpers",
-    "packages/**/docs/**/*",
-    "packages/singleton-manager/demo/",
-    "packages/singleton-manager/test/",
-    // ignore test/demos for singleton manager until overlays are typed as it's used in there
-    "packages-node/**/test-helpers",
-    "packages-node/**/docs/**/*",
-    "packages-node/babel-plugin-extend-docs/**/*.js",
-    "packages-node/providence-analytics/**/*.js",
-    "packages-node/remark-extend/**/*.js"
-  ]
+    "skipLibCheck": true,
+    "incremental": true,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true
+  }
+  // "include": ["packages/**/*.js", "packages-node/**/*.js"],
```

Comment:
should we delete this (and add on package level in future)?

Reply:
deleted